_drafts/release-osiam-3.0.md

```diff
@@ -3,81 +3,124 @@ title:  "Release: OSIAM 3.0"
 ---
 
 We are proud to announce the availability of OSIAM 3.0.
-
-TODO: Write more top level stuff
+This version increases the SCIM 2.0 compliance, contains security enhancements,
+and improves deployment and configuration.
+The literally biggest change is the merge of the auth-server and the resource-server.
+But we also removed custom or outdated APIs, like the Facebook Connect emulation and the PATCH API.
```

Comment:
"But custom and outdated APIs like the Facebook Connect emulation and the PATCH API were also removed"